src/Utils/obiDialogParser.ts

```diff
@@ -30,10 +41,10 @@ export interface ObiDialogParserResult {
 
 export class ObiDialogParser {
     private app: CLM.AppBase
-    //private composerDialog: OBIUtils.ComposerDialog
     private actions: CLM.ActionBase[] = []
     private entities: CLM.EntityBase[] = []
-    private dialogs: OBITypes.OBIDialog[]
+    private dialogs: Map<string, OBITypes.OBIDialog>
```

Comment:
Idk if you just following file convention since there are other `Map`s below, but I thought there was just talk on two previous reviews and only using maps when they're required so people know to watch out.

What prevents this from being `Record<string, OBITypes.OBIDialog>` ?

Reply:
Oh, I just realized how I missed your comments on that other PR.  GitHub doesn't show comments on lines that have changed since a PR has been pushed.  That seems like a horrific bug.

I was following the style of the existing code; updated to dict, similar to other var.